library/swarm

```diff
@@ -0,0 +1,4 @@
+# maintainer: Andrea Luzzardi <al@docker.com> (@aluzzardi)
+# maintainer: Victor Vieux <vieux@docker.com> (@vieux)
+
+latest: git://github.com/docker/swarm-library-image@c0f8204b2a1bb34233cd2144df51a4de9572440c
```

Comment:
For an example of what I mean, see "3.0.0-rc7" on
https://registry.hub.docker.com/_/mongo/ vs
https://registry.hub.docker.com/_/mongo/tags/manage/ which has "3.0.0-rc6"
and "2.8.0-rc5" etc.